gulpfile.js

```diff
@@ -0,0 +1,22 @@
+var gulp     = require('gulp'),
+    mocha    = require('gulp-mocha'),
+    babel    = require('babel'),
+    plumber  = require('gulp-plumber'),
+    istanbul = require('gulp-istanbul');
```

Comment:
I don't like the single var style for what it's worth.

Also, if we're going to be adding more and more to this file, we should invest in one of those plugin autoloader thingamabobs.